### PR TITLE
Add Localytics support, fix example project.

### PR DESCRIPTION
**Pull request by kreeger**

### CODE REVIEW

File: HEAnalytics/HEAnalyticsPlatformLocalytics.swift [hunk 1]
```diff
@@ -0,0 +1,136 @@
+//
+//  HEAnalyticsPlatformLocalytics.swift
+//  EmployeeApp
+//
+//  Created by Ben Kreeger on 3/17/16.
+//  Copyright © 2016 Oven Bits. All rights reserved.
```
Comment (kreeger) [inline HEAnalytics/HEAnalyticsPlatformLocalytics.swift]: Let me change this right quick.

File: HEAnalytics/HEAnalyticsPlatformLocalytics.swift [hunk 2]
```diff
@@ -0,0 +1,163 @@
...
+        }
+        
+        super.initializePlatform(platformData)
+    }
+    
+    
+    /// Has the user opt'd out of data collection? Note this value is not persisted anywhere by HEAnalytics. Exposing this setting in the GUI, persisting the value, restoring the value, and enforcing it generally is the responsibility of the app developer.
+    public override var optOut: Bool {
+        didSet {
+            Localytics.closeSession()
```
Comment (hsoi) [inline HEAnalytics/HEAnalyticsPlatformLocalytics.swift]: What if this gets set to false (they don't want to optOut? maybe they opted out, changed their mind and want to opt back in)?

Comment (kreeger) [inline HEAnalytics/HEAnalyticsPlatformLocalytics.swift]: Great catch. Lemme fix!

**Comment by hsoi** [comment]
I know nothing about Localytics, so I'll have to trust you on all that implementation.

Otherwise, thanx for adding this support!

**Comment by hsoi** [comment]
:+1:

**Pull request merged**